src/Tribe/Views/V2/View.php

```diff
@@ -1940,7 +1940,13 @@ protected function get_ical_data() {
 		 *
 		 * @param string $text The default link text, which is "Export Events".
 		 */
-		$link_text  = apply_filters( 'tribe_events_ical_export_text', __( 'Export Events', 'the-events-calendar' ) );
+		$link_text  = apply_filters(
+			'tribe_events_ical_export_text',
+			sprintf(
+				__( 'Export %s', 'the-events-calendar' ),
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: d1837ae337c874ee4439f72cf47048ee</sub>